install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1819,6 +1819,14 @@ prometheus:
   # ref: https://docs.cilium.io/en/stable/observability/metrics/
   metrics: ~
 
+  # --- Enable controller group metrics for monitoring specific Cilium subsystems. The list
+  # is a list of controller group names. The special values of "all" and "none" are supported.
+  controllerGroupMetrics:
+    - write-cni-file
+    - clustermesh-users-management
```

Comment:
This will not take effect, as the clustermesh-apiserver/kvstoremesh containers do not mount the `cilium-config` ConfigMap. I'd suggest to directly specify the argument (enabling all controllers) in their deployment.

Reply:
One of the concerns with this feature was an increase in metrics cardinality. Given your experience with those components, would you still recommend "all"?

Reply:
I've set both of these to "all" in the Helm chart.

Reply:
Thanks! I'm not personally concerned in case of these two components, as they are very limited in scope, and make use of only a few controllers (mainly the ones used for the etcd client).

Reply:
It could be additionally removed from here, as this controller is not used by the agent.